addons/controller_icons/objects/Sprite2D.gd

```diff
@@ -0,0 +1,27 @@
+@tool
```

Comment:
This should be implemented in the original `Sprite.gd` file instead. It's better to preserve the old file name (but keep the new `ControllerSprite2D` class name), otherwise script references might break if users ports their Godot 3 projects to Godot 4.

Reply:
Sorry I didn't notice this through GitHub's interface, but by testing the changes locally, this file should not be renamed. Rename this file to the original `Sprite.gd` name, otherwise script references will break and users will have to re-add these nodes when porting projects to Godot 4.